arch/src/aarch64/gic.rs

```diff
@@ -101,17 +99,38 @@ pub fn create_gicv3(vm: &VmFd, vcpu_count: u8) -> Result<DeviceFd> {
     Ok(vgic_fd)
 }
 
+// Auxiliary functions for getting addresses and size of where the distributor and redistributor
+// are placed.
+/// Get the address of the GIC distributor.
+pub fn get_dist_addr() -> u64 {
```

Comment:
nit: getters in rust usually omit the `get_` part from their name. I.E. this would just be `dist_addr()`.
Same for the other getters.